contracts/Proxy/TimeOwnedUpgradeabilityProxy.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.5.13;
+
+import { OwnedUpgradeabilityProxy } from "./OwnedUpgradeabilityProxy.sol";
+
+/**
+ * @title TimeOwnedUpgradeabilityProxy
+ * @dev This contract combines an upgradeability proxy with 
+ * basic authorization control functionalities
+ *
+ * This contract allows us to specify a time at which the proxy can no longer
+ * be upgraded
+ */
+contract TimeOwnedUpgradeabilityProxy is OwnedUpgradeabilityProxy {
+
+    uint256 expiration;
+    uint256 constant MAX_UINT = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
+
+    /**
+    * @dev the constructor sets the original owner of the contract to the sender account.
+    */
+    constructor() public {
+        _setUpgradeabilityOwner(msg.sender);
+        expiration = MAX_UINT;
```

Comment:
Can we have this start at 4 months rather than MAX_UINT and set later? I'd like that to be what people see when they look at the code, not MAX_UINT